src/containers/modals/networkPicker/index.test.tsx

```diff
@@ -0,0 +1,82 @@
+import React from 'react'
+import { render, fireEvent } from '@testing-library/react'
+import { Provider } from 'react-redux'
+import CustomThemeProvider from 'themes'
+import configureMockStore from 'redux-mock-store'
+import thunk from 'redux-thunk'
+
+import NetworkPickerModal from './'
+import { mockedInitialState } from 'util/tests'
+
+describe('NetworkPickerModal', () => {
+  it('NetworkPickerModal is Visible', () => {
```

Comment:
```suggestion
 test('Should be visible', () => {
```
As described has component name so in each unit test we don't need to add name again.


If it's unit test let's make use of react testing library with `test` and `it` is part of cypress please take note and do needfull.